packages/codemirror/src/mode.ts

```diff
@@ -125,7 +129,11 @@ namespace Mode {
    */
   export
   function findByFileName(name: string): ISpec {
-    return CodeMirror.findModeByFileName(name);
+    let basename = PathExt.basename(name);
+    if (basename === ""){
```

Comment:
Should I even be doing this check here @blink1073?

Reply:
No, `basename` would only be empty if `name` is also empty.

Reply:
Ah ok sure. Makes sense. Just removed that